pkg/kubectl/cmd/auth/reconcile.go

```diff
@@ -59,6 +59,14 @@ var (
 	reconcileLong = templates.LongDesc(`
 		Reconciles rules for RBAC Role, RoleBinding, ClusterRole, and ClusterRole binding objects.
 
+		Missing objects are created, and the containing namespace is created for namespaced objects, if required.
+		
+		Existing roles are updated to include the permissions in the input objects,
+		and remove extra permissions if --remove-extra-permissions is specified.
+
+		Existing bindings are updated to include the subjects in the input objects,
+		and remove extra subjects if --remove-extra-subjects is specified.
+
 		This is preferred to 'apply' for RBAC resources so that proper rule coverage checks are done.`)
```

Comment:
I know this is existing, but this sounds ominous - as if `apply` bypasses security checks?  Or that `apply` shouldn't be used with RBAC objects?